azuredevops/internal/service/feed/resource_feed.go

```diff
@@ -0,0 +1,136 @@
+package feed
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/feed"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils"
+)
+
+func ResourceFeed() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceFeedCreate,
+		Read:   resourceFeedRead,
+		Update: resourceFeedUpdate,
+		Delete: resourceFeedDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+				Required:     true,
+			},
+			"project_id": {
```

Comment:
Since the feed can be created at org or project level but cannot move between org and project or different projects. `project_id` should be marked with `ForceNew:  true,`

Reply:
updates